src/components/Link/Link.stories.tsx

```diff
@@ -8,10 +8,19 @@ export default {
   parameters: {
     layout: "centered",
   },
-  args: {
-    children: "링크",
-    href: "#",
-  },
+  args: { children: "링크", href: undefined },
+  decorators: [
+    ((Story, context) => {
+      return (
+        <Story
+          args={{
+            ...context.args,
+            href: context.args.href || window.parent.location.href,
+          }}
+        />
+      );
+    }) as Decorator,
```

Comment:
@Sunjae95 타입 안전하지 않은 `as` 쓰시지 마시고 요렇게 깔끔하게 처리하시면 어떨까요?

```suggestion
    (Story, context) => (
      <Story
        args={{
          ...context.args,
          href: context.args.href || window.parent.location.href,
        }}
      />
    ),
```

Reply:
아 그렇군요. 전 다른 방법을 잘 모르겠네요... 그럼 선재님께서 찾으신 방법이 최선이 아닐까요?

Reply:
@Sunjae95  이렇게 하는건 어떠신가요? 
215-link-stroybook-href-bug 브랜치에서 이렇게 변경해서 동작했을때 빌드오류는 안되는걸로 보입니다.
```
const meta: Meta<typeof Link> = {
  component: Link,
  parameters: {
    layout: "centered",
  },
  args: { children: "링크", href: undefined },
  decorators: [
    (Story, context) => {
      return (
        <Story
          args={{
            ...context.args,
            href: context.args.href || window.parent.location.href,
          }}
        />
      );
    },
  ],
};

export default meta;

```